src/mlpack/tests/main_tests/preprocess_imputer_test.cpp

```diff
@@ -120,17 +117,15 @@ TEST_CASE_METHOD(
   }
 
   // Input custom data points and labels.
-  SetInputParam("input_file", (std::string) "preprocess_imputer_test.csv");
+  SetInputParam("input", inputData);
   SetInputParam("missing_value", (std::string) "nan");
   SetInputParam("strategy", (std::string) "listwise_deletion");
-  SetInputParam("output_file",
-      (std::string) "preprocess_imputer_output_test.csv");
+  REQUIRE(params.Get<arma::Row<size_t>>("output").n_cols == inputData.n_cols);
```

Comment:
```suggestion
  REQUIRE(params.Get<arma::mat>("output").n_cols == inputData.n_cols);
```

Output is of type `arma::mat` and not `arma::Row<size_t>`.

Reply:
Thanks